src/lib/functions/netlify-function.mjs

```diff
@@ -177,7 +182,7 @@ export default class NetlifyFunction {
       return false
     })
 
-    return isMatch
+    return match?.pattern
```

Comment:
nit: Returning just the pattern feels a bit random and too tied to the way we're currently using this method. I'd say that it feels more natural to return the whole route and then use from it whatever you need. Totally not a big deal, though.

Reply:
done in [0666c4b](https://github.com/netlify/cli/pull/5970/commits/0666c4b01b382937ad7f13d9d2ce13edc329abec)